### PR TITLE
Updated the routes.js with Prime check

### PR DESCRIPTION
**Pull request by HarshUdai**
First contribution
I have added the logic of prime check, with the help of for loop and a count variable.

Thank you @ZiyanK

### CODE REVIEW

File: routes/routes.js [hunk 1]
```diff
@@ -1,5 +1,20 @@
...
+        if(n%i==0){
+            count=count+1;
+        }
+    }
+    if(count>0){
+        return "Not Prime";
+    }
+    else{
+        return "Prime";
+    }
```
Comment (ZiyanK) [inline routes/routes.js:15]: Kindly shift the logic directly inside the route

File: routes/routes.js [hunk 2]
```diff
@@ -8,4 +23,10 @@ router.post('/palindrome', (req, res) => {
     //Palindrome logic goes here
 })
 
+router.post('/prime/:num', (req, res) => {
+    var num=req.params.num;
```
Comment (ZiyanK) [inline routes/routes.js:27]: Assume the num variable to be inside the body of the request. Please make the necessary change for this.

**Review by ZiyanK — CHANGES_REQUESTED** [review]
Logic looks good to go. I shall merge the code once these 2 changes are made

**Comment by HarshUdai** [comment]
Ok, I will shortly commit new changes.

**Comment by HarshUdai** [comment]
Hey @ZiyanK  i have done the changes required.

**Comment by ZiyanK** [comment]
Good job. Logic looks good. Kindly stick with the appropriate formatting in the future. I shall merge the pr :+1:

**Comment by HarshUdai** [comment]
Thank you, @ZiyanK

**Pull request merged**